src/components/misc/TargetImage.tsx

```diff
@@ -1,22 +1,20 @@
-import { MutableRefObject } from 'react';
+import { HTMLProps, MutableRefObject } from 'react';
 
-import { useTargetRef } from '../../src';
+import { useTargetRef } from '../../hooks';
 
-export function Image({
-  src,
-  style,
-}: {
+interface TargetImageProps extends Omit<HTMLProps<HTMLImageElement>, 'src'> {
   src: string;
-  style?: React.CSSProperties;
-}) {
+}
+
+export function TargetImage(props: TargetImageProps) {
+  const { src, style, ...otherProps } = props;
```

Comment:
the src prop does not need to be extracted and can also be part of the spread props

Reply:
If i do not spread "src" it can be "undefined". I. think, its good to ensure that if we use our TargetImage, src will be everytime passed.